### ISSUE TITLE
Schema's without properties don't return a default

### ISSUE THREAD

**Issue by ninloot** [mnboos/jsonschema-default#7]
I just tried to use the following schema and it didn't create a default object
```
{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "Range",
    "additionalProperties": false,
    "type": "number",
    "default": 0,
    "minimum": -10,
    "maximum": 10
}
```

When I call `jsonschema_default.create_from(<schema>)` the returned object is `{}` instead of the expect `0`.

Looking at the code I see that it's always looking for a properties element. If this the fix is after line 27 add a check for an empty properties and if it is the return call _get_default() passing in the schema instead of the properties. I believe something like this:
```
if not properties:
  return jsonschema_default._get_default(name = '', prop = schema, schema = schema)
```

This use case is a bit trivial but it came up when we added a test to load all the schema in a directory and validate that each generates a default object.

**Comment by mnboos** [comment]
Hi and thank you for your request

Is this a valid schema without `properties`?

**Comment by ninloot** [comment]
I believe it is. I don't think you need to have a properties element.

**Comment by cwiede** [comment]
Related on this, even if the schema represents an object having a "properties" section, then still the default beneath "properties" is ignored, because jsonschema_default only looks for "default" entries below "properties".

**Comment by mnboos** [comment]
You're right. My implementation is incomplete. I'll have to fix that.

**Comment by mnboos** [comment]
So, after [rewriting](https://github.com/mnboos/jsonschema-default/tree/feature/rewrite) the library, it [now works](https://github.com/mnboos/jsonschema-default/commit/375f25c67012f4f2f6c40e2be12f1c0cef5eb553). 

I'll publish the changes soon.

**Comment by mnboos** [comment]
This is now fixed in [release 1.6.0](https://pypi.org/project/jsonschema-default/1.6.0/)
